extensions/Webhooks/template/en/default/admin/params/webhooks.html.tmpl

```diff
@@ -14,7 +14,11 @@
 [%
    param_descs = {
      webhooks_enabled => "Enable or disable the webhooks feature.",
-     webhooks_group => "The name of the group of users who can create webhooks."
+     webhooks_group => "The name of the group of users who can create webhooks.",
      webhooks_any_product_group => "The name of the group of users who can create webhooks for any product."
```

Comment:
Do we need a comma here?

Reply:
They are not required by the template engine, but I added here for consistency.